Dockerfile-helper-worker

```diff
@@ -0,0 +1,23 @@
+FROM python:3.12-slim
+ARG POETRY_VERSION=1.8.2
+
+# set max celery task time to 10 hours
+ENV CELERYD_TIME_LIMIT=36000
```

Comment:
I think we can reduce that to a 15 minutes, job submition doesn't require that long running time.
What is the default time limit ? Maybe default should be enough ?

Reply:
I reduced it to 2 minutes. Should be enough